### PR TITLE
#1954 Added my name to credits.

### LINKED ISSUE #1954: Tanisha Rajgor - Credits
### Description

Add my name to the credits page.

### Acceptance Criteria

See my name in my favorite color on the credits page.

### Proposed Solution

Add my name and favorite color to the "names" array in the CreditsPage.tsx file."

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by tanisharajgor**
## Changes

Added my name to the credits page.

Closes #1954

### CODE REVIEW

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 1]
```diff
@@ -207,7 +207,14 @@ const CreditsPage: React.FC = () => {
...
     { name: 'Will Anderson', color: '#ebb145' },
-    { name: 'Chris Pyle', color: '#ADD8E6' }
+    { name: 'Chris Pyle', color: '#ADD8E6' },
+    {
+      name: 'Tanisha Rajgor',
+      color: '#e3ddfb',
+      sx: {
+        textShadow: '0 0 2px rgba(0, 0, 0, 0.4), 0 0 5px rgba(180, 160, 255, 0.8)'
+      }
+    }
```
Comment (Aaryan1203) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:217]: Great work!

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
You want to remember to fill out the PR template. That includes a description of what you did as well as a screenshot of the website with your changes

**Comment by tanisharajgor** [comment]
> You want to remember to fill out the PR template. That includes a description of what you did as well as a screenshot of the website with your changes

Apologize about that, here is the updated description:

**Description:**
Added my name to the credits page, with a light purple color and white, muted glow effect. 

**Fullscreen View**:
Fullscreen

**Smallest Window View**:
SmallestWindow

Closes https://github.com/Northeastern-Electric-Racing/FinishLine/issues/1954

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**